client/src/components/LayerPlayer.jsx

```diff
@@ -27,33 +27,44 @@ export default function LayerPlayer({ layers, trackId, userId, recordingHandler,
   const playAllLayers = async () => {
     try {
       await Tone.start();
+      await Tone.loaded();
+
       let keys = Object.keys(playerStore.allPlayers)
       keys.forEach((layerKey, i) => {
-        let layer = playerStore.allPlayers[layerKey]
-        layer.start()
+
         Tone.Transport.schedule((time) => {
           Tone.Draw.schedule(() => {
             renderWaveform(layer.waveform, layerKey);
           }, time);
         }, "+0.005");
+        let layer = playerStore.allPlayers[layerKey]
```

Comment:
Might have found a bug here. Will line 37 know what 'layer' is if it isn't defined until line 40?

Reply:
Oddly enough it still seems to know. Possibly because the transport is async and will have access to it by the time it starts, but I'll move it back up anyway.